src/pages/RegisterStaff.jsx

```diff
@@ -0,0 +1,82 @@
+import { RegisterStaffForm } from '@/forms/staff/forms';
+import { registerStaffSchema } from '@/forms/staff/schemas';
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from '@/components/ui/card';
+import { registerStaff } from '@/services/staff';
+import { useState } from 'react';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useForm } from 'react-hook-form';
+import { useNavigate } from 'react-router';
+
+export function RegisterStaff() {
+    
+  return (
+    <div className="flex min-h-screen items-center justify-center">
+      {/* Right side - Register Staff Form */}
+      <div className="w-full lg:w-2/3 flex items-center justify-center p-8">
+        <RegisterStaffCard />
+      </div>
+    </div>
+  );
+}
+
+function RegisterStaffCard() {
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState('');
+  const navigate = useNavigate();
+
+  const form = useForm({
+    resolver: zodResolver(registerStaffSchema),
+    defaultValues: {
+      email: '',
+      password: '',
+      name: '',
+      surname: '',
+      specialty: '',
+      dni: '',
+    },
+  });
+
+  function onSubmit(values) {
+    setIsLoading(true);
+    registerStaff(values)
+      .then((response) => {
+
+        if (response.status === 201)
+          navigate('/app');
+      })
+      .catch((err) => {
+        
+        if (err.response && err.response.status === 400) {
+          setError('A doctor with the same DNI or Email already exists');
+        } else if (err.response && err.response.status === 401) {
+          setError('You must be a clinic admin to register staff');
+        } else {
+          setError('An error occurred. Please try again later.');
+          console.error(err);
+        }
+        setIconsRed();
+      })
+      .finally(() => {
+        setIsLoading(false);
+      });
+  }
+
+  return (
+    <Card className="w-full max-w-md min-w-[300px] rounded-lg shadow-sm">
+      <CardHeader className="items-start">
+        <CardTitle>Register Staff</CardTitle>
+        <CardDescription>Register new staff member</CardDescription>
+      </CardHeader>
+      <CardContent>
+        <RegisterStaffForm form={form} onSubmit={onSubmit} isLoading={isLoading} error={error} />
+      </CardContent>
+    </Card>
+  );
+}
+
+function setIconsRed() {
+  document.querySelectorAll('.pulse').forEach((div) => {
+    div.classList.add('error-pulse');
+    div.classList.remove('pulse');
+  });
+}
```

Comment:
Remove

Reply:
Resolved in https://github.com/FIS2425/frontend/pull/30/commits/c373c58ffad46f3a3a7951d71f15f2925651791d